### PR TITLE
Add setting and credentials to Awepay

### PR DESCRIPTION
**Pull request by dara123**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/Awepay.yaml [hunk 1]
```diff
@@ -17,6 +17,18 @@ allOf:
             description: Awepay rcode.
             format: password
             writeOnly: true
+          secretKey:
+            type: string
+            description: Awepay secret key for P2P REST API.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/Awepay.yaml:22]: nitpick:
```suggestion
            description: Awepay secret key for the P2P REST API.
```

File: openapi/components/schemas/GatewayAccountConfig/Awepay.yaml [hunk 2]
```diff
@@ -17,6 +17,18 @@ allOf:
...
         required:
           - sid
           - rcode
+      settings:
+        type: object
+        description: Awepay setting.
+        properties:
+          useP2pRest:
+            type: boolean
+            description: Specifies whether to use P2P REST API.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/Awepay.yaml:34]:
```suggestion
            description: Specifies whether to use the P2P REST API.
```

**Review by noon-dawg — APPROVED** [review]
Looks good on the docs side

File: openapi/components/schemas/GatewayAccountConfig/Awepay.yaml [hunk 3]
```diff
@@ -17,6 +17,18 @@ allOf:
...
+            format: password
+            writeOnly: true
         required:
           - sid
           - rcode
+      settings:
+        type: object
+        description: Awepay setting.
+        properties:
+          useP2pRest:
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Awepay.yaml:32]: Camel cased `P2p` looks strange to me..

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**